packages/@angular/cli/tasks/eject.ts

```diff
@@ -287,7 +287,7 @@ class JsonWebpackSerializer {
         return v;
       } else if (typeof v == 'string') {
         if (v === path.join(this._root, 'node_modules')) {
-          return this._serializeRegExp(/\/node_modules\//);
+          return this._relativePath('process.cwd()', 'node_modules');
```

Comment:
Totally makes sense this way... Will try it locally and then do the requested change